### DISCUSSION TITLE
Option to change font

### DISCUSSION THREAD

**Discussion by Recht-robots** [Akifyss/obsidian-border#110] [Q&A] [answered]
Hi everyone,

First off, I absolutely love this theme! Used quite a lot of themes, but this one stuck around as my main one. However, after looking everywhere in the style settings, I cannot find an option to change the font for text, only headings. Am I overlooking it or is that option not available yet? If not available, would there be any other way to change the font?

Thanks in advance!

**Comment by Akifyss** [comment, marked as answer]
Currently, there are no more font options available in style settings, but you can change text font and interface font here:
<img width="1100" alt="image" src="https://github.com/Akifyss/obsidian-border/assets/23567154/797f8278-5aed-4bce-aaa5-4bb4f4b30249">

**Reply by Recht-robots** [reply]
Ah damn, completely overlooked that. Thank you so much for the quick answer and amazing work!
